lib/index.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+import 'babel-polyfill';
+import {} from 'dotenv';
```

Comment:
Oops, no it does not.  Though it would if I fixed it to use the following.

```js
import {} from 'dotenv/config'
```

However as dotenv is being preloaded (which is why it works currently) it can get removed altogether.